src/main/kotlin/com/jobtracker/api/controllers/models/ContactModel.kt

```diff
@@ -0,0 +1,24 @@
+package com.jobtracker.api.controllers.models
+
+import com.jobtracker.api.repository.entities.ApplicationEntity
+import com.jobtracker.api.repository.entities.ContactEntity
+import com.jobtracker.api.repository.entities.CompanyEntity
+import org.apache.commons.lang3.mutable.Mutable
+import java.util.UUID
+
+data class ContactModel(
+  var contactId: UUID = UUID.randomUUID(),
```

Comment:
this will need to be last

Reply:
sure - no problem.
What's the impact of that though out of curiosity?